middleware/golfer.go

```diff
@@ -33,6 +33,7 @@ func GolferHandler(next http.Handler) http.Handler {
 				)  SELECT u.admin,
 				          COALESCE(u.country, ''),
 				          u.delete,
+						  u.discord,
```

Comment:
the alignment here looks wrong, it's tab indent up to the sql, then spaces, sorry about that :joy: